docs/sources/installation/ubuntulinux.md

```diff
@@ -1,13 +1,13 @@
-page_title: Installation on Ubuntu 
-page_description: Instructions for installing Docker on Ubuntu. 
```

Comment:
I see other headers don't have a trailing space, so I think it's "safe" to remove